src/github_changelog/github.clj

```diff
@@ -14,39 +14,45 @@
   (let [{:keys [github-api user repo]} config]
     (format "%s/repos/%s/%s/pulls" (strip-trailing github-api "/") user repo)))
 
-(defn- make-request [config params]
-  (let [oauth-token (:token config)]
-    {:as           :json
-     :query-params (merge {:state "closed"} params)
-     :headers      {"User-Agent"    "GitHub-Changelog"
-                    "Authorization" (str "token " oauth-token)}}))
-
-(def call-api (throttle-fn http/get 5 :second))
+(defn- make-request
+  ([config] (make-request config {}))
+  ([config params]
+   (let [oauth-token (:token config)]
```

Comment:
You could remove this let, if you use destructuring in the `fn` argument definition:
```clojure
([{oauth-token :token} params]
 ...
```
(This applies to a lot of `fn`s in this namespace actually (See above.)).